### ISSUE TITLE
Installation fails on Windows

### ISSUE THREAD

**Issue by yaskevich** [JoshuaWise/integer#19]
Win7, x64
```
D:\app>npm i integer

> integer@2.1.0 install D:\app\node_modules\integer
> node-gyp rebuild


D:\app\node_modules\integer>if not defined npm_config_node_gyp (node "C:\Users\user\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\user\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "C:\Users\user\.windows-build-tools\python27\python.exe", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (C:\Users\user\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (C:\Users\user\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\configure.js:509:16)
gyp ERR! stack     at callback (C:\Users\user\AppData\Roaming\npm\node_modules\npm\node_modules\graceful-fs\polyfills.js:289:20)
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:153:21)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\user\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd D:\app\node_modules\integer
gyp ERR! node -v v10.16.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN sqltest@1.0.0 No description
npm WARN sqltest@1.0.0 No repository field.
```

**Comment by zangornjak** [comment]
There's an error message in your log. It can't find the Python executable in `"C:\Users\user\.windows-build-tools\python27\python.exe"`. Put it there or set the PYTHON `env` variable.

**Comment by cekvenich** [comment]
Let me know if this fixes it before the install: npm i -g node-gyp@latest

**Comment by JoshuaWise** [comment]
This should be fixed with `integer 3.0.0` due to the use of prebuilt binaries.

**Comment by JoshuaWise** [comment]
Try upgrading to version 4.0.0
